NEWS.md

```diff
@@ -7,16 +7,16 @@ Since the last stable release many things have happened:
 - jq now lives at <https://github.com/jqlang>
 - New maintainers, admins, and owners have been recruited.
   - A list of [current maintainers](https://github.com/jqlang/jq/blob/70bbd10b0b58e797d03963264fc934879bb44454/AUTHORS#L4-L13)
-- CI, scan builds, release builds etc now use GitHub actions. @owenthereal #2596 #2620
+- CI, scan builds, release, website builds etc now use GitHub actions. @owenthereal @itchyny #2596 #2603 #2620 #2723
```

Comment:
Can probably add my name to this also :)

Reply:
Of course!